Sources/Views/Cells/ContactMessageCell.swift

```diff
@@ -28,7 +28,7 @@ open class ContactMessageCell: MessageContentCell {
     
     public enum ConstraintsID: String {
         case initialsContainerLeftConstraint
-        case disclouserRigtConstraint
+        case disclosureRigtConstraint
```

Comment:
`Rigt` vs `Right`

Reply:
Thanks for helping me. This is my first stab at contributing to any open source project. I'm on a solo mobile dev team so somethings are still new to me!

That typo is also resolved now.